x-pack/plugins/aiops/public/components/log_rate_analysis/log_rate_analysis_results.tsx

```diff
@@ -341,6 +350,40 @@ export const LogRateAnalysisResults: FC<LogRateAnalysisResultsProps> = ({
           />
         </EuiFlexItem>
       </ProgressControls>
+      {showLogRateAnalysisResultsTable && (
+        <>
+          <EuiSpacer size="s" />
+          <EuiCallOut
+            title={
+              <span data-test-subj="aiopsAnalysisTypeCalloutTitle">
+                {currentAnalysisType === LOG_RATE_ANALYSIS_TYPE.SPIKE
+                  ? i18n.translate('xpack.aiops.analysis.analysisTypeSpikeCallOutTitle', {
+                      defaultMessage: 'Analysis type: Log rate spike',
+                    })
+                  : i18n.translate('xpack.aiops.analysis.analysisTypeDipCallOutTitle', {
+                      defaultMessage: 'Analysis type: Log rate dip',
+                    })}
+              </span>
+            }
+            color="primary"
+            iconType="pin"
+            size="s"
+          >
+            <EuiText size="s">
+              {currentAnalysisType === LOG_RATE_ANALYSIS_TYPE.SPIKE
+                ? i18n.translate('xpack.aiops.analysis.analysisTypeSpikeCallOutContent', {
+                    defaultMessage:
+                      'The median log rate in the selected deviation time range is higher than the baseline. Therefore, the results shows statistically significant items within the deviation time range that are contributors to the spike. The "doc count" column refers to the amount of documents in the deviation time range.',
```

Comment:
`the results show` or being consistent with the style used for dips - `the analysis results table shows`

Reply:
Updated in 4f8c71ac387.